packages/react/src/components/ProductDataList/src/ProductDataList.stories.tsx

```
@@ -0,0 +1,312 @@
+        <ProductDataList.Section
+          content={
+            <Box display="flex" gap="2">
+              {radios.map((radio) => (
+                <Radio
+                  as="button"
+                  name={radio.name}
+                  value={radio.value}
+                  label={radio.label}
+                  checked={selected === radio.value}
+                  onChange={() => setSelected(radio.value)}
+                />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)